Sources/TransifexNative/Core.swift

```diff
@@ -23,6 +23,64 @@ public enum TXRenderingStategy : Int {
     case icu
 }
 
+/// Bypasses swizzling and offers the bundled translation for the passed locale, if found.
+///
+/// If the string is not found the logic returns the value of the `params` dictionary and if this is
+/// also not found, it returns the `sourceString`.
+final class BypassLocalizer {
+    let bundle : Bundle?
+    
+    /// Initializes bypass localizer with a certain locale.
+    ///
+    /// - Parameter localeCode: The locale to be used
+    init(with localeCode: String) {
+        /// Get the bundle for the provided locale code, if it exists.
+        guard let bundlePath = Bundle.main.path(forResource: localeCode,
+                                                ofType: "lproj") else {
+            self.bundle = nil
+            return
+        }
+
+        self.bundle = Bundle(path: bundlePath)
+    }
+    
+    private func extractBundledString(sourceString: String,
+                                      params: [String: Any]) -> String? {
+        guard let bundle = bundle else {
+            return nil
+        }
+        
+        let tableName = params[Swizzler.PARAM_TABLE_KEY] as? String
+        
+        // We use the SKIP_SWIZZLING_VALUE constant to skip swizzling for this
+        // call, so that the actual value can be retrieved, if it exists in the
+        // application bundle.
+        let localizedString = bundle.localizedString(forKey: sourceString,
+                                                     value: Swizzler.SKIP_SWIZZLING_VALUE,
+                                                     table: tableName)
+        
+        if  localizedString != Swizzler.SKIP_SWIZZLING_VALUE {
+            return localizedString
+        }
+  
+        return nil
+    }
+    
+    func get(sourceString: String,
+            params: [String: Any]) -> String {
```

Comment:
Didn't check, but it looks like this could fit into 1 line.

Reply:
True, it fits!